### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by ShaydeNofziger**

### CODE REVIEW

**Comment by ShaydeNofziger** [comment]
Unrelated test case seems to be failing.

**Comment by jarrydlee** [comment]
@ShaydeNofziger the failing test has been removed in PR #31

File: public/javascripts/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function setUsername(){
```
Comment (jarrydlee) [inline public/javascripts/index.js]: This can be removed now that we wont be mocking it, correct?

File: public/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+								</td>
+							</tr>
+					</tbody>
+				</table>
+			</div>
+		</div>
+</div>
+
+<script src="/javascripts/index.js" />
+<script>setUsername();</script>
```
Comment (jarrydlee) [inline public/templates/index.html]: This should also be removed

**Comment by ShaydeNofziger** [comment]
Creating a front-end branch from which all front-end implementations will be checked out from / merged to prior to a master merge.

**Pull request closed without merging**